app/containers/Scatter/reducer.js

```diff
@@ -5,27 +5,28 @@
  */
 
 import { fromJS } from 'immutable';
-import { SCATTER_LOADED, EOSCLIENT_LOADED, ATTACHED_ACCOUNT, DETACHED_ACCOUNT } from './constants';
+import { SCATTER_LOADED, EOSCLIENT_LOADED, ATTACHED_ACCOUNT, DETACHED_ACCOUNT, REFRESHED_DATA } from './constants';
```

Comment:
looks too long, make sure to run `yarn run format:js`, should be fixed

Reply:
I did run it :(

Reply:
Haha all good then!! I thought this was longer than 120 characters, my bad :)